app/src/main/java/com/atom/android/muvik/ui/favorite/FavoriteFragment.kt

```diff
@@ -0,0 +1,34 @@
+package com.atom.android.muvik.ui.favorite
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.fragment.app.Fragment
+import com.atom.android.muvik.base.BaseFragment
+
+import com.atom.android.muvik.databinding.FragmentFavoriteBinding
+
+class FavoriteFragment : BaseFragment<FragmentFavoriteBinding>(FragmentFavoriteBinding::inflate) {
+    override fun initData() {
+
```

Comment:
Chỗ nào hàm trống sau ông thêm /*TODO implement later*/ nhé, hoặc //not support nếu hàm đấy ko triển khai. Không sau có con detekt kiểm tra convention kĩ lắm